docs/index.md

```diff
@@ -134,7 +134,11 @@ resource "hcp_vault_cluster" "example" {
 Required:
 
 - `resource_name` (String) The resource_name of the Workload Identity Provider to exchange the token with.
-- `token_file` (String) The path to a file containing a JWT token retrieved from an OpenID Connect (OIDC) or OAuth2 provider.
+
+Optional:
+
+- `token` (String) The JWT token retrieved from an OpenID Connect (OIDC) or OAuth2 provider. Exactly one of `token_file` or `token` must be set.
```

Comment:
What is the motivation of doing an exactly one?

Reply:
This is two different ways of specifying the same information. Either the token has been written to disk, in which case you should specify the `token_file` variable, or the token has been loaded dynamically and Terraform just has it, in which case you should specify it directly via `token`.

It doesn't make sense to specify both of these together. Similarly, you must specify the token somehow if you are authenticating via the block, so you must specify at least one.

Reply:
I've changed this to `AtLeastOneOf`, and the `token` field takes precedence over the `token_file` field.